memory/main.py

```diff
@@ -948,8 +948,8 @@ def item_address(num):
 
 def get_items_order():
     items = []
-    for x in range(100):
-        items.append(process.read_bytes(item_address(x), 1))
+    for x in range(150):
+        items.append(process.readBytes(item_address(x), 1))
```

Comment:
All of these should be updated to snake case. Otherwise they are not going to work.